pkg/client/preflight.go

```diff
@@ -97,3 +99,18 @@ func preflightVersionCheck(client kubernetes.Interface) error {
 
 	return nil
 }
+
+func preflightExistingSonobuoy(client kubernetes.Interface, cfg *PreflightConfig) error {
+	_, err := client.CoreV1().Pods(cfg.Namespace).Get("sonobuoy", metav1.GetOptions{})
+	switch {
+	// Pod doesn't exist: great!
+	case apierrors.IsNotFound(err):
+		return nil
+	case err != nil:
+		return errors.Wrap(err, "error checking for Sonobuoy pod")
+	// No error: pod exists
+	case err == nil:
+		return errors.New("sonobuoy run already exists in this namespace")
```

Comment:
probably out of scope for this, but what do you think about giving the user some action on a failed preflight? for instance in this case, either delete the pod or run it in a different namespace

Reply:
Yeah, I thought about that. It doesn't feel right to put that sort of thing inside of a error message, though. Maybe at some point we could make an ErrorWithSolution() type that had both an `Error()` and a `Solution()` method on it, then the CLI could check for the latter?

Reply:
yeah!